### PR TITLE
add sbt to differ accounts

### PR DESCRIPTION
**Pull request by RustemYuzlibaev**
### **Description**

- **What does this PR do?**
  - _A brief explanation of the change, its purpose, and how it impacts the codebase._
- **Why is this change needed?**
  - _Describe the problem the PR solves or the feature it adds._

### **Type of change**

- [ ] Bug fix
- [ ] New feature
- [ ] Refactor (cleaning up code without changing functionality)
- [ ] Documentation update
- [ ] Tests
- [ ] Other (describe below):

---

### **Checklist**

- [ ] Code adheres to coding guidelines and standards.
- [ ] Linting and formatting checks have passed, no local issues.
- [ ] Existing tests pass successfully, no local errors.
- [ ] Pull request is linked to a relevant issue or task.
- [ ] Tests are written for new features or fixes.
- [ ] Documentation has been updated (if applicable).

---

### **Related Issue/Task**

- Issue/Task link: _[Paste link to issue or Jira ticket]_

---

### **Testing**

- **How has this been tested?**
  - _Describe manual testing steps or link to relevant automated tests._
- **Edge cases covered:**
  - _List any specific cases handled by this PR._

---

### **Screenshots/Video/Link to dedicated environment (if applicable)**

- _Include any visual examples of the change + dedicated environment (if there is a sort of automation)._

---

### **Additional Notes**

- _Add any information the reviewer should consider, especially cross-project impacts, UX changes, or potential risks._

---

### **Reviewer Tasks**

- [ ] Review logic for correctness and clarity.
- [ ] Verify that the code is following best practices.
- [ ] Test or review related areas if this PR affects other projects.

### CODE REVIEW

File: packages/sdk/src/presto/index.ts [hunk 1]
```diff
@@ -113,6 +114,7 @@ export class PrestoModule<T> {
    */
   public async createCropReceipt(
     amount: string,
+    profit: number,
```
Comment (wer1st) [inline packages/sdk/src/presto/index.ts:117]: Is it processed as percentage?

**Review by wer1st — APPROVED** [review]
